Documentation/ApiOverview/BackendLayout/Index.rst

```diff
@@ -0,0 +1,211 @@
+
+
+.. _be-layout-definition:
+
+Backend Layout Definition
+=========================
+
+Backend Layouts can be configured either as "Backend Layout" record in a sysfolder or as PageTsConfig entry in
+ :typoscript:`mod.web_layout.BackendLayouts`. Each layout will be saved with a key. The Backend Layout records are
+using their uid as a key, therefore layouts defined via PageTsConfig should use a non-numeral String key. It is a good
+practise to use a descriptive name as key.
+
+The entries title and icon are being used to display the Backend Layout options in the page properties.
+
+The overall grid size will be defined by :typoscript:`config.backend_layout.colCount` and :typoscript:`rowCount`.
+Additional rows in the :typoscript:`rows` Array and additional columns in the each rows :typoscript:`columns` section
+will be ignored when they are greater then :typoscript:`rowCount` or :typoscript:`colCount` respectively.
+
+Each column position can span several columns and or several rows. Each column position must have a distinct number
+between 0 and n assigned. It is best practise to always assign "0" to the main column if there is such a thing as a
+main column. Multiple Backend Layouts that contain similar parts, i.e. header, footer, aside, ... should within one
+project have the same number assigned each. This simplifies sliding up to find content within one column position to
+use content of a parent page across layouts.
+
+
+.. _be-layout-simple-example:
+
+Backend Layout Simple Example
+=============================
+
+The following PageTsConfig example creates a simple Backend Layout consisting of two rows and just one Columns.
+
+.. code-block:: typoscript
+
+   mod {
+     web_layout {
+       BackendLayouts {
+         exampleKey {
+           title = Example
+           config {
+             backend_layout {
+               colCount = 1
+               rowCount = 2
+               rows {
+                 1 {
+                   columns {
+                     1 {
+                       name = LLL:EXT:frontend/Resources/Private/Language/locallang_ttc.xlf:colPos.I.3
+                       colPos = 3
+                       colspan = 1
+                     }
+                   }
+                 }
+                 2 {
+                   columns {
+                     1 {
+                       name = Main
+                       colPos = 0
+                       colspan = 1
+                     }
+                   }
+                 }
+               }
+             }
+           }
+           icon = EXT:example_extension/Resources/Public/Images/BackendLayouts/default.gif
+         }
+       }
+     }
+   }
+
+.. _be-layout-advanced-example:
+
+Backend Layout Advanced Example
+===============================
+
+The following PageTsConfig example creates a 3x3 Backend Layout with 5 column position sections in total. The topmost
+row, the "Header" spans all 3 columns. There is an "Aside" spanning two rows on the right.
```

Comment:
I would write: "The topmost row, here called "Header" spans over all of the 3 columns.